t/ui/14-dashboard-parents.t

```diff
@@ -146,5 +146,28 @@ subtest 'filtering subgroups' => sub {
     isnt(scalar @{$driver->find_elements('opensuse test', 'link_text')}, 0, "child group 'opensuse test' present'");
 };
 
+subtest 'View grouped by group' => sub {
+    my $parent_group_id = $schema->resultset('JobGroupParents')->search(undef, {name => 'Test parent'})->first->id;
```

Comment:
you should be able to replace `search(undef, {name => 'Test parent'})->first` with `find(…)`

Reply:
... because the column `name` has a unique constraint. And even if you wanted to use search the search criteria should be the first parameter (or are you doing something very special here?).

Reply:
No, I don't think is special. Changing this